src/content/releases/_index.en.md

```diff
@@ -1,10 +1,26 @@
 +++
-date = 2016-04-09T16:50:16+02:00
+date = 2020-05-04T16:50:16+02:00
 title = "Releases"
 pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.3.0 Lighthouse Service Discovery without KubeFed
+
+> This release is focused on removing the KubeFed dependency from Lighthouse, improving the user experience
+> and adding experimental WireGuard support as an alternative to IPsec
```

Comment:
Review on github is hard if we don't break the lines because then you can't comment (or propose suggestions) individually in single lines... it becomes about the whole paragraph. 

Also diffs are worse, because it's removing a whole paragraph to add a new paragraph when it's just may be a word.

I was going to propose line length limit checking on commits.

Reply:
Actually I think review is harder when text is broken on separate lines b/c many times proposed changes extend to subsequent line(s).  So when you do a suggested change, you have to do it on multiple lines which is awkward b/c github doesn't allow you to select multiple lines to edit.

Reply:
Actually it does let you select multiple lines and propose suggestions on them even.

As for simplicity of review on GH thats more of an individual preference I guess?
Myself, I find that it doesn't bother me if it's in multiple lines or not.

I agree about shell diff (which is poor), but with rich diff (such as GH though they're quite bad at it) it's really easy to see exactly what changed.

My gripe with this is mainly that the source looks one way and then you expect the output to look similarly, but it gets rendered differently (and probably rightly so).

Reply:
> Actually it does let you select multiple lines and propose suggestions on them even.
> 

Ahh you're right - just figured that out.  That helps a lot  - I'm OK with multiple lines now 😄